portfolio/src/main/webapp/comments.js

```diff
@@ -13,24 +13,120 @@
 // limitations under the License.
 
 /**
- * Fetches comments from the server and adds it to the DOM.
+ * An object to manage the query string of a URL
+ * @const {URLSearchParams}
  */
-function getComments() {
-  fetch('/comments').then(response => response.json()).then((comments) => {
-    const commentsContainer = document.getElementById('comments-container');
+const GET_PARAMS = new URLSearchParams ({
+  imageId: '',
+  quantity: '5',
+  dateOrder: 'true'
+});
 
-    commentsContainer.innerHTML = '';
-    for (const comment of comments) {
-      commentsContainer.appendChild(createListElement(comment));
+/**
+ * Delete all the comments
+ */
+function deleteAllComments(imageId) {
+  const postParams = new URLSearchParams();
+  postParams.append('imageId', imageId);
+
+  fetch('/delete-comments', {method: 'POST', body: postParams}).then(response => {
+    if (response.status === 200) {
+      getComments();
+    } else {
+      alert('Sorry, we could not process your request. ' +
+          'Reload the page and try again.');
     }
   });
 }
 
-/** Creates an <li> element containing text. */
-function createListElement(text) {
-  const liElement = document.createElement('li');
-  liElement.innerText = text;
-  return liElement;
+/**
+ * Fetches comments from the server and adds them to the DOM.
+ */
+function getComments() {
+  fetch(`/list-comments?${GET_PARAMS.toString()}`).then((response) => {
+    if (response.status === 200) {
+      response.json().then((comments) => {
+        const commentsContainer = document.querySelector('#comments-container');
+        
+        removeNodeChildren(commentsContainer);
```

Comment:
Maybe call this removeCommentsFromDom()? That way the naming matches up with the addCommentsToDom() function.

Reply:
Sounds good, I used this kind of generic name for it, because I thought I might use it for another container, but now I don't think I will, so I will change it.

Reply:
Done!